cdpacctest/acctest.go

```diff
@@ -126,3 +150,103 @@ func GetCdpClientForAccTest() *cdp.Client {
 	})
 	return cdpClient
 }
+
+type awsAccountCredentials struct {
+	name          string
+	accountID     string
+	externalID    string
+	defaultPolicy string
+}
+
+func NewAwsAccountCredentials(name string) *awsAccountCredentials {
```

Comment:
same here

Reply:
I've exported the type because I had to rebase and push becaue of the linter changes anyways